src/util.rs

```diff
@@ -48,6 +48,27 @@ pub fn history() -> web_sys::History {
 pub fn html_document() -> web_sys::HtmlDocument {
     wasm_bindgen::JsValue::from(document()).unchecked_into::<web_sys::HtmlDocument>()
 }
+/// Convenience function to access the `web_sys::HtmlCanvasElement`.
+pub fn canvas(id: &str) -> web_sys::HtmlCanvasElement {
+    let canvas = document().get_element_by_id(id).unwrap();
+    canvas
+        .dyn_into::<web_sys::HtmlCanvasElement>()
+        .map_err(|_| ())
+        .unwrap()
+}
```

Comment:
I'd rather return `Option<HtmlCanvasElement>` or `Result` and write comment when the returned value is `None`. 
Example:
https://github.com/David-OConnor/seed/blob/f21c80c460349823e5658cc8655f532f4771830e/src/util.rs#L54
Unwraps / expects make sense for document or body, but I don't trust the user with the correct `id`.